collections/src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -266,6 +284,19 @@ trait IterableOps[+A, +CC[X], +C] extends Any {
     */
   def to[C1](factory: Factory[A, C1]): C1 = factory.fromSpecific(toIterable)
 
+  def toList: immutable.List[A] = immutable.List.from(toIterable)
+
+  def toVector: immutable.Vector[A] = immutable.Vector.from(toIterable)
+
+  def toMap[T, U](implicit ev: A <:< (T, U)): immutable.Map[T, U] =
```

Comment:
Minor remark about naming: we should try to stick to the same conventions everywhere. `Map` names its type parameter `K` and `V`, we should reuse the same names here.